packages/pwa/app/routes.jsx

```diff
@@ -81,26 +76,31 @@ const routes = [
         exact: true
     },
     {
-        path: '/:locale/cart',
+        path: '/:siteIdAlias?/:locale/cart',
         component: Cart,
         exact: true
     },
     {
-        path: '/:locale/product/:productId',
+        path: '/:siteIdAlias?/:locale/product/:productId',
         component: ProductDetail
     },
     {
-        path: '/:locale/search',
+        path: '/:siteIdAlias?/:locale/search',
         component: ProductList
     },
     {
-        path: '/:locale/category/:categoryId',
+        path: '/:siteIdAlias?/:locale/category/:categoryId',
         component: ProductList
     },
     {
-        path: '/:locale/account/wishlist',
+        path: '/siteIdAlias?/:locale/account/wishlist',
         component: Wishlist
     },
+    {
+        path: '/:siteIdAlias?/:locale',
```

Comment:
need to move this route to the bottom so as it is too broad, it can match with some cases above (e.g /en-GB/cart)